### PR TITLE
test: moving tests to dedicated repository

### PR DESCRIPTION
**Pull request by hugoseabra-imachines**
Reorganized tests to a dedicated directory, as it was causing cyclic depedencies.

### CODE REVIEW

**Comment by vkomodey** [comment]
@hugoseabra-imachines please, sign the CLA for CI

**Comment by vkomodey** [comment]
> Reorganized tests to a dedicated repository

I suppose you mean "dedicated file", not a "dedicated repository", am I correct?

File: bin/shell.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+set -exu
+CMD=${1-'./bin/wait_then_run'}
```
Comment (vkomodey) [inline bin/shell.sh]: Can't see the usage of this CMD

File: bin/wait_then_run [hunk 2]
```diff
@@ -26,11 +26,9 @@ done
 
 mkdir -p minio/escrow-results
 
-export PYTHONPATH=$(pwd)
+#export PYTHONPATH=$(pwd)
```
Comment (vkomodey) [inline bin/wait_then_run]: Let's delete commented code instead of keeping it

File: hmt_escrow/storage.py [hunk 3]
```diff
@@ -25,7 +25,8 @@
 
 ESCROW_BUCKETNAME = os.getenv("ESCROW_BUCKETNAME", "escrow-results")
 ESCROW_AWS_ACCESS_KEY_ID = os.getenv("ESCROW_AWS_ACCESS_KEY_ID", "minio")
-ESCROW_AWS_SECRET_ACCESS_KEY = os.getenv("ESCROW_AWS_SECRET_ACCESS_KEY", "minio123")
+ESCROW_AWS_SECRET_ACCESS_KEY = os.getenv("ESCROW_AWS_SECRET_ACCESS_KEY",
+                                         "minio123")
```
Comment (vkomodey) [inline hmt_escrow/storage.py]: just imho, 80 characters in one line is something in the past. 120 chars seem fine for all modern devices

**Review by vkomodey — CHANGES_REQUESTED** [review]

**Comment by hugoseabra-imachines** [comment]
> @hugoseabra-imachines please, sign the CLA for CI

I have signed it. 3 times! :)

image

**Comment by hugoseabra-imachines** [comment]
> > Reorganized tests to a dedicated repository
> 
> I suppose you mean "dedicated file", not a "dedicated repository", am I correct?

The correct is "dedicated directory". All tests was sent to directory `test/` in *root*. My bad.

File: hmt_escrow/storage.py (hunk 3 above)
Comment (hugoseabra-imachines) [reply hmt_escrow/storage.py]: Alright.

Comment (hugoseabra-imachines) [reply hmt_escrow/storage.py]: I did not change the code due to number of chars to not cause many changes. But I have configured my environment locally to 120 chars.

**Review by vkomodey — APPROVED** [review]

**Comment by hugoseabra-imachines** [comment]
> > @hugoseabra-imachines please, sign the CLA for CI
> 
> I have signed it. 3 times! :)

Something is wrong with license/cla. I did sign a couple of times and it is still pending.

Did you guys have this issue before?

**Comment by vkomodey** [comment]
@hugoseabra-imachines I see that for commits you are using @hugoseabra account instead of @hugoseabra-imachines . That might be the confusion
Please, make sure @hugoseabra-imachines signed the CLA

**Comment by vkomodey** [comment]
@hugoseabra-imachines 
sorry, one last thing here - could you please increase the library version? a minor update in package.json and  setup.py - 0.10.1

**Comment by hugoseabra-imachines** [comment]
> @hugoseabra-imachines sorry, one last thing here - could you please increase the library version? a minor update in package.json and setup.py - 0.10.1

- [x] Done!

**Pull request merged**